src/tokenlist.schema.json

```diff
@@ -176,7 +176,6 @@
         "chainId": {
           "type": "string",
           "description": "The chain ID of the Ethereum network where this token is deployed",
-          "minimum": 1,
```

Comment:
removed this because this does not seem to align with new implementation and I don't think we use it anywhere except it kept failing during test run